.github/workflows/ci.yml

```
@@ -27,13 +27,6 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
```

If this works we can keep the cache for the hex packages, adding the otp-version to the cache key, and skip using an optional restore-key when the key was not found
```suggestion

      - name: Cache Hex packages
        uses: actions/cache@v2
        with:
          path: ~/.cache/rebar3/hex/hexpm/packages
          key: ${{ runner.os }}-${{ matrix.otp-version }}-hex-${{ hashFiles('**/rebar.lock') }}
```

---

This should avoid using the the same built packages for all otp-versions